mrjob/dataproc.py

```diff
@@ -225,6 +205,12 @@ def __init__(self, **kwargs):
         """
         super(DataprocJobRunner, self).__init__(**kwargs)
 
+        # check for library support
+        if google is None:
+            raise ImportError(
+                'You must install google-cloud and google-cloud-dataproc'
```

Comment:
Good point.

Reply:
You know, I think I'll do the opposite. The point of this warning is so that the user knows generally what the problem is (no library installed), rather than getting a cryptic `AttributeError`.

I'm a bit iffy about pinning `google-cloud-dataproc` to a particular version. I've found other libraries that do this to be nuisance. I think instead I'll just have `setup.py` allow any version (since version 0.1.0 works), and warn people that they might need to upgrade the library in later versions because it's in alpha.